yt/frontends/ytdata/data_structures.py

```diff
@@ -629,7 +629,7 @@ def get_data(self, fields=None):
         fluids, particles = [], []
         finfos = {}
         for ftype, fname in fields_to_get:
-            finfo = self.ds._get_field_info(ftype, fname)
+            finfo = self.ds._get_field_info((ftype, fname))
```

Comment:
another spot where it splits the tuple only to re-assemble.

Reply:
done